.kokoro/presubmit/system.cfg

```diff
@@ -11,3 +11,6 @@ env_vars: {
     key: "PYTEST_ADDOPTS"
     value: "-n=auto --dist=loadscope"
 }
+
+# Kokoro VM timeout of 5 hours for system tests
+timeout_mins: 300
```

Comment:
What's the justification for this? The default is 180.

Reply:
It's to match the continuous system run https://github.com/googleapis/python-aiplatform/blob/main/.kokoro/continuous/system.cfg.